modules/EnsEMBL/Web/Component/Gene/ComparaOrthologs.pm

```diff
@@ -423,7 +424,8 @@ sub get_export_data {
 
     if (keys %ok_species) {
       # It's the lower case species url name which is passed through the data export URL
-      return [grep {$ok_species{lc($hub->species_defs->production_name_mapping($_->get_all_Members->[1]->genome_db->name))}} @$homologies];
+      my $lookup = $hub->species_defs->prodname_to_url_lookup;
+      return [grep {$ok_species{lc($lookup->{$_->get_all_Members->[1]->genome_db->name})}} @$homologies];
```

Comment:
Is `$ok_species` above always going to be lowercased?

Reply:
I suspect that lc can be removed, but it doesn't seem to be hurting anything for now!